WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -7,22 +7,41 @@ You do not need to download the blockchain for this setup!
 
 ## Setup Bitcoin Knots with RegTest
 
-Bitcoin Knots is working very similarly to Bitcoin Core. You can get a grasp with [this guide](https://bitcoin.org/en/developer-examples).
+BCurrently, the latest version(25.1) is not tested, so the preferred version is [23.0](https://github.com/bitcoinknots/bitcoin/releases/tag/v23.0.knots20220529).
 
 Todo:
 
-1. Install [Bitcoin Knots](https://bitcoinknots.org/) on your computer. Verify the PGP - there is a tutorial [here](https://bitcoinknots.org/)
-2. Start Bitcoin Knots with: bitcoin-qt.exe -regtest then quit immediately. In this way the data directory and the config files will be generated.
+1. Install [Bitcoin Knots 23.0](https://bitcoinknots.org/files/23.x/23.0.knots20220529/) on your computer. Verify the PGP - there is a tutorial [here](https://bitcoinknots.org/)
+2. Start Bitcoin Knots with: bitcoin-qt.exe -regtest then quit immediately. In this way the data directory and the config files will be generated. If you write "-datadir" parameter,make sure the directory exists
+
+     Windows x64:
+    ```
+    "C:\Program Files\Bitcoin\bitcoin-qt.exe" -regtest -blockfilterindex -txindex -datadir=c:\Bitcoin
+    ```        
+    macOS: 
+    ```
+    "/Applications/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt" -regtest -blockfilterindex -txindex -datadir=$HOME/Library/Application Support/Bitcoin"
+    ```
+    Linux: 
+    ```
+     ~/bitcoin-[version number]/bin/bitcoin-qt -regtest -blockfilterindex -txindex -datadir=c:\Bitcoin
```

Comment:
> -datadir=c:\Bitcoin

AFAIK there is no C directory in Linux, so this should be corrected.

Reply:
Yes, there definitely isn't. Usually either `$HOME/.bitcoin` (default, no need to specify) or `/var/lib/bitcoind` is used. Same applies also to macOS too, although I'm not sure about exact paths there. `C:\` is Windows specific.